components/sidebar/sidebar_category/sidebar_category_menu/sidebar_category_menu.tsx

```diff
@@ -75,12 +83,17 @@ const SidebarCategoryMenu = (props: Props) => {
         }
 
         deleteCategoryMenuItem = (
-            <Menu.ItemAction
-                isDangerous={true}
+            <Menu.Item
                 id={`delete-${props.category.id}`}
+                isDestructive={true}
                 onClick={handleDeleteCategory}
-                icon={<TrashCanOutlineIcon size={16}/>}
-                text={formatMessage({id: 'sidebar_left.sidebar_category_menu.deleteCategory', defaultMessage: 'Delete Category'})}
+                leadingElement={<TrashCanOutlineIcon size={18}/>}
+                labels={
+                    <FormattedMessage
+                        id='sidebar_left.sidebar_category_menu.deleteCategory'
+                        defaultMessage='Delete Category'
+                    />
+                }
```

Comment:
in this case we should add `()`

```suggestion
                labels={(
                    <FormattedMessage
                        id='sidebar_left.sidebar_category_menu.deleteCategory'
                        defaultMessage='Delete Category'
                    />
                )}
```

Reply:
changed